ruby/rails/security/audit/xss/avoid-link-to.yaml

```diff
@@ -0,0 +1,37 @@
+rules:
+- id: avoid-link-to
+  metadata:
+    owasp: 'A6: Security Misconfiguration'
+    cwe: "CWE-279: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')"
+    references:
+    - https://brakemanscanner.org/docs/warning_types/link_to/
+    - https://brakemanscanner.org/docs/warning_types/link_to_href/
+    category: security
+    technology:
+    - rails
+  message: |
+    This code includes user input in `link_to`. In Rails 2.x, the body of `link_to` is not escaped.
```

Comment:
I'm confused about the "values starting with `javascript:` or `data:` ..." statement (copied from brakeman), did not see this reflected in the brakeman rule? Is this about the user input? What should they do here?

Reply:
Yes, `javascript:` and `data:` are the possible values of user input that can be malicious, so the message is correct here :+1: